astro/src/content/blog/understanding-the-authorization-code-grant.mdx

```diff
@@ -0,0 +1,127 @@
+
+{/* more */}
+
+<YouTube id="mmyfwovnago" />
+
+## What Is The Authorization Code Grant?
+
+The Authorization Code Grant is a widely used method in OAuth 2.0 for enabling users to securely authenticate with third-party applications. For the tech nerds in the house that is [RFC6749](https://datatracker.ietf.org/doc/html/rfc6749). This method involves redirecting the user to an authorization server, obtaining an authorization code, and then exchanging that code for an access token. 
+Let's break this process down step-by-step.
+
+### Starting Point: The Initial URL
+
+Our journey begins by navigating to the admin application for FusionAuth. Initially, the URL [https://sandbox.fusionauth.io/admin](https://sandbox.fusionauth.io/admin) points to FusionAuth's admin UI, which then delegates authentication to FusionAuth itself. This delegation redirects us to the hosted login page that is both themed and customizable.
+
+<Aside type="note">
+FusionAuth allows deep [customization](/docs/customize/look-and-feel/), so the look and feel can differ based on the client's branding.
+</Aside>
+
+### Customized Login Page
+
+In the below example, we'll use a customized login page from one of our clients as an example. You will notice familiar elements such as email and password fields, but there's a distinct branding as intended by Private Division. Once logged in, we proceed through the authentication flow and get redirected to the Admin UI.
+
+
+![Private Division's custom login page with video game highlights on the left and a login form on the right.](/img/blogs/video-auth-code-grant/1-video-auth-code-grant.png)
+
+#### Why the Redirect is Crucial
+
+The redirect serves two main purposes: security and features.
+
+1. **Security**:
+    
+    
+    - By redirecting users to FusionAuth's hosted login pages, we make sure that applications delegating authentication to FusionAuth don't need to handle sensitive information like credentials.
+    - This minimizes the risk of sensitive data being spread across multiple applications.
+    
+    <Aside type="note">
+    The application simply sends the user to FusionAuth. FusionAuth performs the checks and then returns the necessary information to the application.
+    </Aside>
+2. **Features**:
+    
+    
+    - Delegating authentication to FusionAuth enables us to offer new features without burdening your application.
+    - Features like [profile](/docs/lifecycle/manage-users/account-management/updating-user-data) management, [Multi-Factor Authentication (MFA)](/docs/lifecycle/authenticate-users/multi-factor-authentication), [session management](/docs/lifecycle/authenticate-users/logout-session-management), etc., can be added through FusionAuth or its APIs with minimal complexity.
+    
+    <Aside type="note">
+    New features can be added via FusionAuth or its APIs, keeping the application blissfully unaware of the complexities involved.
+    </Aside>
```

Comment:
Could point here for further detail: https://fusionauth.io/docs/get-started/core-concepts/hosted-login-vs-api-login

Reply:
Since we aren't doing the hosted in this video / sdk's I think it would be better to leave that one out.